src/Components/WebAssembly/testassets/HostedInAspNet.Client/wwwroot/seleniumworkaround.js

```diff
@@ -0,0 +1,24 @@
+(function () {
```

Comment:
Nit: is it worth considering having one shared place for identical files like this js file and copy it to each of the test folders by reference instead of having multiple copies?

Reply:
I started off with that, but it looks like we rely on being able to serve these files from the testassets's physical dir. We could do something to copy them over as part of build, but this felt cheap enough to me to start with.

Reply:
Personally I'm fine either way on this, based on the idea that this is hopefully quite temporary.

Might be nice to add a comment in the file like:

```suggestion
// Note: there are multiple copies of this file throughout the repo. If you're editing it, please look for
// other seleniumworkaround.js files and keep them all in sync.

(function () {
```